src/components/ListV2/ListCell.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled'
+import { MouseEventHandler, ReactNode } from 'react'
+
+const StyledCellDiv = styled.div<{ colSpan?: number }>`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  min-height: 60px;
+  ${({ colSpan }) => (colSpan ? `grid-column: span ${colSpan} / span 12;` : '')}
+`
```

Comment:
Can be a Stack?

Reply:
No because of colSpan prop is not allowed by Stack